src/routes/All.js

```diff
@@ -9,6 +9,7 @@ export default (props) => {
     <Router>
       <Switch>
         <Route exact path="/" component={HomePage} />
+        <Route exact path="/login" component={HomePage} />
```

Comment:
Since now home page is the same with login page, it's now simply directed to the same component.

Reply:
/lgtm